server/models/Price.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+const validator = require("validator")
+
+const priceSchema = new Schema({
+  game: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Game",
+    require: true,
+  },
+  merchant: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Merchant",
+    required: true,
+  },
+  price_text: {
```

Comment:
this should probably just be price, gets confusing...and redudant. or just text...but that seems to generic

Reply:
Will change to price for now.